engine/src/main/java/org/terasology/logic/location/LocationComponent.java

```diff
@@ -72,12 +73,18 @@ public void setLocalPosition(Vector3f newPos) {
         position.set(newPos);
     }
 
+    /**
+     * @return The direction this entity is facing, relative to any parent
+     */
     public Vector3f getLocalDirection() {
         Vector3f result = Direction.FORWARD.getVector3f();
         getLocalRotation().rotate(result, result);
         return result;
     }
 
+    /**
+     * @return The location of this component, relative to any parent
```

Comment:
Is it location or rotation? And what is the difference between direction and rotation?

Reply:
Prexisting code, but direction would be a vector pointing the way the entity is facing, while rotation is a quaternion I imagine.